app/helpers/dropdown_helper.rb

```diff
@@ -15,6 +15,30 @@ def organisation_options_for_select(selected = nil)
     options_for_select(organisation_options, selected)
   end
 
+  def topic_options_for_select(selected = nil)
+    topic_option_attributes = ->(topic) do
```

Comment:
(I personally think `lambda` is less confusing syntax, more in keeping with Ruby's block syntax)

Reply:
Addressed in 098a0615cb1972e6c27b6bdcdc2bedf6686a8d31
cc/ @andrewgarner